tests/models/test_image.py

```diff
@@ -442,6 +441,7 @@ def test_multiple_images_badness(
                 im.save(no_archive=True)
                 filenames.append(im.get_fullpath(as_list=True)[0])
                 images.append(im)
+                im = im.recursive_merge( session )
```

Comment:
Why do we need these merges now? Does it have to do with the exposure's provenance?